### PR TITLE
Mongodb service guide

### PR DESCRIPTION
**Pull request by viivek46**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by SarahStegall** [comment]
Please open a JIRA ticket for this PR and link to it.

File: pages/services/mongodb-enterprise/0.1.0-4.0.3/quick-start-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+For Installing MongoDB OpsManager on a host machine, please gather the requirements specified and follow the steps provided on the [MongoDB OpsManager Installation](https://docs.opsmanager.mongodb.com/current/installation/) page.
+
+
+## Install MongoDB Service on DC/OS
+
+MongoDB can be installed via either the DC/OS Catalog web interface or by using the CLI. 
+
+Below are the steps to install MongoDB using the DC/OS Catalog Web Interface:
+
```
Comment (SarahStegall) [inline pages/services/mongodb-enterprise/0.1.0-4.0.3/quick-start-guide/index.md]: There are no steps, no text, only a few screencaps. Please add steps explaining what users see in the screen grabs and what actions they need to take.

**Comment by ryadav88** [comment]
Docs review Jira: https://jira.mesosphere.com/browse/DCOS-47246

File: pages/services/mongodb-enterprise/0.1.0-4.0.3/quick-start-guide/index.md (hunk 1 above)
Comment (AnandJat) [reply pages/services/mongodb-enterprise/0.1.0-4.0.3/quick-start-guide/index.md]: Hi @SarahStegall
Required steps and text are added.
cc : @ryadav88

**Comment by SarahStegall** [comment]
The directory under which these docs are presented is /mongodb-enterprise/. However, these docs are marked "enterprise: false" in all the metadata. Furthermore, the landing page states that "MongoDB is an open-source document database". Why is the word "enterprise" included in the directory name? And if there is no good reason to retain it, may I rename the directory to be in conformance with Mesosphere naming standards?

File: package.json [hunk 2]
```diff
@@ -1,78 +0,0 @@
-{
```
Comment (niartenyaw) [inline package.json]: Hi @viivek46 you have deleted this `package.json` file, please restore it.

**Comment by vibhujain** [comment]
Hi @niartenyaw The changes has been made as per your suggestions. Please review the guide.
Also, I request you to please have a look at the PR raised for staging.
Ref: https://github.com/mesosphere/dcos-docs-site/pull/1894
cc @viivek46 @AnandJat

**Comment by niartenyaw** [comment]
Thanks for fixing that. Looks good from my end. @SarahStegall will be reviewing the content for you (or tag someone who will).

@SarahStegall let me know when you're ready to merge and i can help with conflicts

**Comment by vibhujain** [comment]
Thanks for your review @niartenyaw . We'll follow up with @SarahStegall to finish our review.

**Comment by SarahStegall** [comment]
I am adding a Do Not Merge and duplicate label to this PR because it has been superseded by [PR1894](https://github.com/mesosphere/dcos-docs-site/pull/1894). Please direct all comments and edits to that PR.

**Comment by ck4adventure** [comment]
Closing as dup.

**Pull request closed without merging**